packages/rainbowkit/src/themes/darkTheme.ts

```diff
@@ -58,6 +58,7 @@ export const darkTheme = ({
     profileForeground: 'rgba(224, 232, 255, 0.05)',
     selectedOptionBorder: 'rgba(224, 232, 255, 0.1)',
     standby: '#FFD641',
+    connected: '#00E676',
```

Comment:
We already have `connectionIndicator` for chains; let's use that color and styling key instead